### ISSUE TITLE
[BUG/ISSUE] GEOS-Chem fails on GFAS_pFe when enabling GFAS emissions

### ISSUE THREAD

**Issue by lizziel** [geoschem/HEMCO#173]
## Describe the bug:
GC-Classic and GCHP 14.0 both fail with the same error in HEMCO if enabling GFAS biomass burning emissions. 

### Expected behavior: ###

GEOS-Chem should run to completion if GFAS is enabled.

### Actual behavior: ###

Running out-of-the-box 14.0 benchmark simulation, but with GFAS set to `true` in `HEMCO_Config.rc`, results in the following error:
Program received signal SIGSEGV: Segmentation fault - invalid memory reference.

```
Backtrace for this error:
#0  0x2b0ca9f433ff in ???
#1  0x2b0ca90e6ee9 in sflush
        at ../.././libgfortran/io/unix.h:89
#2  0x44f6af in __emissions_mod_MOD_emissions_run
        at /n/home08/elundgren/testruns/gcc_gfas_pfe/ref.gc_4x5_merra2_fullchem_benchmark/CodeDir/src/GEOS-Chem/GeosCore/emissions_mod.F90:203
#3  0x405963 in geos_chem
        at /n/home08/elundgren/testruns/gcc_gfas_pfe/ref.gc_4x5_merra2_fullchem_benchmark/CodeDir/src/GEOS-Chem/Interfaces/GCClassic/main.F90:1481
#4  0x403b96 in main
        at /n/home08/elundgren/testruns/gcc_gfas_pfe/ref.gc_4x5_merra2_fullchem_benchmark/CodeDir/src/GEOS-Chem/Interfaces/GCClassic/main.F90:32
Segmentation fault (core dumped)
```

### Error messages

The log file with `VERBOSE` and `WARNINGS` set to 3 in `HEMCO_Config.rc` contains this:
```
Successfully updated diagnostics: EmispFe_Anthro (counter:  1)
HEMCO: Entering GET_CURRENT_EMISSIONS (hco_calc_mod.F90) ( 3)

 Evaluate field GFAS_pFe

HEMCO ERROR: Illegal altitude unit
 --> LOCATION: GetIdx (hco_calc_mod.F90)

HEMCO ERROR: ERROR 17
 --> LOCATION: GetVertIndx (HCO_CALC_MOD.F90)

HEMCO ERROR:  Error getting vertical index at location            1          46 : GFAS_pFe

HEMCO ERROR: ERROR 6
 --> LOCATION: HCO_CalcEmis (HCO_CALC_MOD.F90)

HEMCO ERROR: ERROR 3
 --> LOCATION: HCO_RUN (HCO_DRIVER_MOD.F90)
===============================================================================
GEOS-Chem ERROR: Error encountered in "HCO_Run"!
 -> at HCOI_GC_Run (in module GeosCore/hco_interface_gc_mod.F90)

THIS ERROR ORIGINATED IN HEMCO!  Please check the HEMCO log file for
additional error messages!
===============================================================================
```

### Additional context

I am able to make the simulation run to completion if I change the GFAS section of `HEMCO_Config.rc` from this...
```
0 GFAS_SO2   $ROOT/GFAS/v2018-09/$YYYY/GFAS_$YYYY$MM.nc so2fire       2003-2021/1-12/1-31/0 C xyL=1:scal300 kg/m2/s SO2  75       5 3
0 GFAS_pFe   -                                          -             -                     - -             -       pFe  75/66    5 3
```
...to this...
```
0 GFAS_SO2   $ROOT/GFAS/v2018-09/$YYYY/GFAS_$YYYY$MM.nc so2fire       2003-2021/1-12/1-31/0 C xyL=1:scal300 kg/m2/s SO2  75       5 3
0 GFAS_pFe   $ROOT/GFAS/v2018-09/$YYYY/GFAS_$YYYY$MM.nc so2fire       -                     - -             -       pFe  75/66    5 3
```

### Your HEMCO version and runtime environment:

 - HEMCO version: **3.5.0**
 - Compiler version: **gfortran10.2**
 - Did you run on a computational cluster, on the AWS cloud: **Harvard Cannon**
 - Are you using GEOS-Chem "out of the box" (i.e. unmodified): **no**
   - If you have modified GEOS-Chem, please list what was changed: **Change GFAS from false to true**

**NOTE: I am able to reproduce this error using both GEOS-Chem Classic and GCHP**

### Input and log files to attach

 - HEMCO.log: **output sent to gc.log**
 - HEMCO_Config.rc: 
[HEMCO_Config.rc.txt](https://github.com/geoschem/HEMCO/files/9814107/HEMCO_Config.rc.txt)

 - Any other error messages: 
[gc.log.txt](https://github.com/geoschem/HEMCO/files/9814106/gc.log.txt)

**Comment by lizziel** [comment]
The above quick fix is only for GC-Classic. For GCHP you need to duplicate the line for GFAS_SO2 in both HEMCO_Config.rc and ExtData.rc, changing SO2 to pFe in both.

**Comment by yantosca** [comment]
We can close this issue since PR #1621 has been merged into the GEOS-Chem 14.1.1 development stream.
